CMakeLists.txt

```
@@ -317,11 +323,18 @@ if (USE_GNUTLS)
 	target_link_libraries(datachannel PRIVATE GnuTLS::GnuTLS)
 	target_link_libraries(datachannel-static PRIVATE GnuTLS::GnuTLS)
 	if (NOT NO_WEBSOCKET)
-		# Needed for SHA1, it should be present as GnuTLS cryptography backend
+		# Needed for SHA1, it should be present as GnuTLS/MbedTLS cryptography backend
 		find_package(Nettle REQUIRED)
 		target_link_libraries(datachannel PRIVATE Nettle::Nettle)
 		target_link_libraries(datachannel-static PRIVATE Nettle::Nettle)
 	endif()
+elseif(USE_MBEDTLS)
+	find_package(MbedTLS CONFIG REQUIRED)
```

```suggestion
	find_package(MbedTLS REQUIRED)
```
I don't think `CONFIG` is needed.

A finder should be added to the project since MbedTLS, since neither CMake or MbedTLS do and it might help you for checking for needed components (e.g. X509). 

Here [one](https://github.com/PatTheMav/obs-studio/blob/macos-build-update/cmake/finders/FindMbedTLS.cmake), you could use. It's the one plan to be used OBS Studio in the future, some OBS repo related comments could be removed.

---

Side-note if there any issues with the finder don't hesitate to report them there: https://github.com/obsproject/obs-studio/pull/8247

---

@RytoEX Fixed! Thanks for calling that out.